plugins/generic/lucene/locale/pt_PT/emailTemplates.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../../../../lib/pkp/dtd/emailTemplateData.dtd">
```

Comment:
Too many `../`s here too.

Reply:
From the US template `<!DOCTYPE email_texts SYSTEM "../../../../../lib/pkp/dtd/emailTemplateData.dtd">`

Reply:
again, I might be sleeping

Reply:
Nope, you're absolutely right! Need more tea.